packages/frontend/core/src/pages/workspace/detail-page/detail-page.tsx

```diff
@@ -131,6 +133,12 @@ const DetailPageImpl = memo(function DetailPageImpl() {
     return;
   }, [doc, globalContext, isActiveView, mode]);
 
+  useEffect(() => {
```

Comment:
is it necessary to execute in every render?

Reply:
It only needs to be executed once. The ignored dependency list is a mistake, I have updated the code.